### PR TITLE
allow non-wrap responses from hw-transport-u2f

### PR DESCRIPTION
**Pull request by flocks**
### Context

I ran into a situation where the firmware, by mistake,  didn't add the 5 bytes before the actual response. This is the case for the apdu `f1d0000000` which returns the current scrambleKey. You can set a scrambleKey to null ( `\00` ) and ask for the current scrambleKey with `f1d0000000`. It basically allow us to know which app is currently open.

### CODE REVIEW

**Comment by gre** [comment]
i wonder if 

- no scramble key should be allowed (we simply skip the XOR logic). this is if i'm correct the equivalent of such a `\0` scramble key
- in such case we switch to your "unwrap" mode 🤔  or would it be too magically? I mean, maybe no need to introduce a new method if we can solve it like this

File: packages/hw-transport-u2f/src/TransportU2F.js [hunk 1]
```diff
@@ -54,6 +55,9 @@ function attemptExchange(
     const { signatureData } = response;
     if (typeof signatureData === "string") {
       const data = Buffer.from(normal64(signatureData), "base64");
+      if (!unwrap) {
+        return data.toString("hex");
```
[the remark this answers is not in the document]
Comment (flocks) [reply packages/hw-transport-u2f/src/TransportU2F.js]: oups sorry just rebased, and the debug is back

**Comment by flocks** [comment]
Yes you are right, I set the scrambleKey to `\0`. 

I don't know if it would be too "magical" yes. 

The thing is I don't know if there are other cases where a null scrambleKey is needed and where the result gets *correctly* wrapped with 5 u2f bytes. But I talked to Oto who clearly said to me that he didn't wrap it by mistake.

**Comment by gre** [comment]
we'll go like this for now

**Pull request merged**